recipes/ubuntu.rb

```diff
@@ -18,3 +18,7 @@
   ignore_failure true
 end
 
+package 'ghostscript' do
```

Comment:
Is this specific to base, or meteor base? It looks like a thing we want for the app. If that's the case, we should put it in the premera-ops cookbook in the default recipe. Base means everything including the buildserver. But if we need it in the buildserver (as part of the meteor build process) it belongs here.